client/hotrod-client/src/test/java/org/infinispan/client/hotrod/AuthenticationTest.java

```diff
@@ -65,10 +66,15 @@ public void testAuthenticationViaURI() {
 
    @Test(expectedExceptions = TransportException.class)
    public void testAuthenticationFailWrongAuth() {
-      ConfigurationBuilder clientBuilder = newClientBuilder();
-      clientBuilder.security().authentication().callbackHandler(new TestCallbackHandler("user", "realm", "foobar"));
-      remoteCacheManager = new RemoteCacheManager(clientBuilder.build());
-      remoteCacheManager.getCache();
+      try {
```

Comment:
Should revert these?

Reply:
Ah, yes. Let me double check and revert everything in the client-hotrod module. Guess I missed some.